README.md

```diff
@@ -1,10 +1,6 @@
 ![kraken-js](public/krakenLogo.png)
 
-
-kraken.js
-=========
-
-Lead Maintainer: [Jean-Charles Sisk](https://github.com/jasisk)
```

Comment:
@jasisk the sentiment is that we shouldn’t have repo maintainers, it puts pressure on those maintainers. We are moving towards a generalist maintainer model!